css/menu.css

```diff
@@ -26,7 +26,7 @@ body.has-menu-left.has-push-menu {
 html.no-android body.has-push-menu,
 html.no-android.has-push-menu {
   -webkit-overflow-scrolling: auto;
-  overflow: visible;
+  overflow-y: hidden;
```

Comment:
This line is for iOS to override the behaviors declared in the lines above, where `.has-push-menu` is specified to se the body with `overflow: hidden`.

So, I don't think the solution is to add `overflow-y: hidden` like you did. Instead, it would be to remove both properties set for `html.no-android body.has-push-menu, html.no-android.has-push-menu {	html.no-android.has-push-menu`.

Reply:
I re-read my last message and realised that it didn't make very much sense.

The lines above the changes you made showed that the property `overflow: hidden` is already applied to `body.has-menu-right.has-push-menu` and `body.has-menu-left.has-push-menu`. Essentially it's applied to `body.has-push-menu` because I assume it always has either one of `.has-menu-left` or `.has-menu-right` applied.

So, with `overflow: hidden` already applied, why is it that you need to apply `overflow-y: hidden`?

The line you've changed here with `overflow-y: visible` is meant to override the `overflow: hidden` previously applied. So, if you want `overflow-y hidden`, you should just be looking to delete the entire block of:

```css
html.no-android body.has-push-menu,	html.no-android body.has-push-menu,
html.no-android.has-push-menu {	html.no-android.has-push-menu {
  -webkit-overflow-scrolling: auto;	  -webkit-overflow-scrolling: auto;
  overflow: visible;
}
```